src/cmd/rest_test.go

```
@@ -83,6 +83,37 @@ func TestCORSPreflightAllowsUIHeaders(t *testing.T) {
+		config.McpOAuthIssuerURL = previousIssuer
+	})
+
+	app := fiber.New()
+	app.Use(newCORSMiddleware())
+	app.Get("/gowa/oauth/authorize", func(c fiber.Ctx) error {
+		return c.SendStatus(fiber.StatusOK)
+	})
+
+	req := httptest.NewRequest("GET", "/gowa/oauth/authorize", nil)
+	req.Header.Set("Origin", "https://client.example.com")
+	resp, err := app.Test(req)
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)